README.md

```diff
@@ -1,17 +1,120 @@
+How are we to interact with our computer _as developers_? While you might have
+been using computers for years, you've likely been engaging with them as a
+_user_ not a _developer_.
 
-The video below describes what the "native" shell (by which we mean the one that comes with the computer) looks like on Mac OS X. 
+That's fine, computers have probably helped you accomplish a great many things,
+but you've only been working with them in the ways that were provided _by other
+developers_. If you wanted to change the behavior, to make a mouse action or a
+keyboard key behave differently, you've not been able the make that change.
+Similarly, when one of the applications locked up or crashed your machine,
+you've not had the ability to look "underneath the hood," to borrow a metaphor
+from automobiles, to take a look and theorize about what changes to make.
+You've lacked a way for "talking" to the computer in a less-mediated form.
+
+Developers "talk" to the computer and the operating system by engaging it at
+the shell level. We use a program called a "shell" to be able to ask questions
+of the processing chip, of the memory, of the disk drives, etc. This interface
+might feel a bit "old-school" or clunky at first, but power users of the shell
+are truly something to behold. With a few keystrokes they can change millions
+of files' names and process them to create thumbnails per each in a new
+directory with but a few short words. They can find hung programs and tell them
+to close in such a way that the computer _recovers_ and doesn't require reboot.
+
+Shell skills is a critical skill in being a developer and your familiarity and
+comfort with it will signal to other developers your professional credentials.
+
+## Learning Objectives
+
+1. Define what a shell program is
+2. Recognize a shell interface
+3. Recognize operating systems providing a shell
+
+## Define What a Shell Program Is
+
+The Shell is the program on your computer that takes in commands you type and
+takes action based on those commands. You can think of yourself as having a
+conversation with the computer.
+
+When you see a shell it will be at a "prompt." This means, from the computer's
+perspective "I'm listening......" While prompts may look different from
+computer to computer (they're customizable and can hold the data you find
+helpful: date, time, name of directory you're in, etc.), they all have the same
+essential function of ferrying your command to the operating system and running
+it.
+
+You can ask the computer to do a great variety of tasks: some or simple file
+manipulation that you might recognize as soemthing you do with a mouse: moving
+a file, renaming a file, making a folder (or subdirectory, as shell-masters
+call it).
+
+You can ask the computer to run programs like `ruby my_awesome_program.rb` or
+even to launch an application!
+
+You can even ask the computer to do data analytics and data collation work
```

Comment:
We're not really doing things like this in this particular course - should we be giving this as an example?